template/koa-rest-api/src/tracing.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * OpenTelemetry tracing initialisation. This is a standalone TS/JS module that is not
+ * referenced by application source code directly. It is required at runtime using the
+ * node command's `--require` argument, see Dockerfile for details.
+ */
+
+import { propagation } from '@opentelemetry/api';
+import { CompositePropagator } from '@opentelemetry/core';
+import { CollectorTraceExporter } from '@opentelemetry/exporter-collector-grpc';
+import { AwsInstrumentation } from '@opentelemetry/instrumentation-aws-sdk';
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+import { B3InjectEncoding, B3Propagator } from '@opentelemetry/propagator-b3';
+import { NodeSDK } from '@opentelemetry/sdk-node';
+
+const app = 'opentelemetry';
```

Comment:
For posterity, I was wondering where this `app` log field convention was from and it looks like our sidecar follows suit.

Reply:
Old mate `ca-go-library` 🙈

`seek-auth-sidecar` was originally `ca-auth-sidecar` and used the default logger from `ca-go-library`. The newer `name` convention comes from Pino/Bunyan.